.gitignore

```diff
@@ -8,3 +8,4 @@ debug/
 **/*.rs.bk
 crt.pem
 spin.sig
+spininstall/
```

Comment:
Probably a good safety net here but `spininstall` dir should already be deleted once Spin is installed [as part of the script](https://github.com/suneetnangia/wasm-orchestration-with-spin/blob/331823c6ca712304103a2b644c294231e7680130/.devcontainer/bootstrap.sh#L32C7-L32C29).

Reply:
The problem was that I cancelled the bootstrap due to some circumstances. Wanted to have it on the safe side.
Also added the parameter for mkdir -p spininstall to create the directory in case it some how exists or was not clean-up automatically because I cancelled the run of the script for some reason

Reply:
Wonder if we should instead have a clean slate in this instance i.e. delete the existing directory and create it fresh, keeping the old dir with unknown state (via -parents flag) may create problem for re-runs? Regardless, keeping `spininstall` in gitignore seems to be fine.